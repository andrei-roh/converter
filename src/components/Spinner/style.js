import styled from 'styled-components';

export const Edge = styled.div`
  display: flex;
  flex-direction: row;
  justify-content: space-evenly;
  align-items: flex-start;
  box-sizing: border-box;
  width: 100%;
  height: 100vh;
`;

export const Wheel = styled.div`
  align-self: center;
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
  z-index: 2;
`;
