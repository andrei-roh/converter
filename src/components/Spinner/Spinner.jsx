import React from 'react';
import Loader from 'react-loader-spinner';
import { BigBlock, SmallBlock } from './style';

const Spinner = () => {
  return (
    <BigBlock>
      <SmallBlock>
        <Loader type="Watch" color="#0064EB" height={250} width={250} />
      </SmallBlock>
    </BigBlock>
  );
};

export default Spinner;
