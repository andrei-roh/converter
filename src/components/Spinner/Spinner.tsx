import Loader from 'react-loader-spinner';
import { BigBlock, SmallBlock } from './style';

const Spinner = () => {
  return (
    <BigBlock>
      <SmallBlock>
        <Loader type="Oval" color="#3f51b5" height={250} width={250} />
      </SmallBlock>
    </BigBlock>
  );
};

export default Spinner;
