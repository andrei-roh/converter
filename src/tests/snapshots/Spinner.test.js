import { shallow } from 'enzyme';
import Spinner from '../../components/Spinner/Spinner';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Spinner />);
});

describe('<Spinner /> rendering', () => {
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
