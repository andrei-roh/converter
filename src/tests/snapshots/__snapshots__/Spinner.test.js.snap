// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> rendering renders correctly 1`] = `ShallowWrapper {}`;
